### DISCUSSION TITLE
How to see mounted volumes on the Desktop?

### DISCUSSION THREAD

**Discussion by brskl88** [lxqt/lxqt#2060] [Q&A]
Hey everybody.I have been using Lubuntu 20.04 and it has lxqt 0.14.I can not find any option to enable to show mounted partitions on the desktop.How can i see my mounted partitions on the Desktop automatically?

**Comment by tsujan** [comment]
So far, no one has been interested in implementing it (→ https://github.com/lxqt/pcmanfm-qt/issues/1192) but we accept PRs.
